infrastructures/docker/src/main/java/org/eclipse/che/workspace/infrastructure/docker/DockerMachineStarter.java

```diff
@@ -650,11 +669,12 @@ protected void setNonExitingContainerCommandIfNeeded(ContainerConfig containerCo
 
     // Inspect container right after start to check if it is running,
     // otherwise throw error that command should not exit right after container start
-    protected void checkContainerIsRunning(String container) throws IOException, InfrastructureException {
+    protected ContainerInfo checkContainerIsRunning(String container) throws IOException, InfrastructureException {
```

Comment:
Maybe it's better to change method name to `getRunningContainer` or something else, wdyt?